roles/nv_gpu_install_from_commit/tasks/build.yml

```diff
@@ -0,0 +1,73 @@
+---
+- name: find a node with our tuned label
+  command: oc get nodes -l "{{ tuned_module_fuse_label }}" -oname
+  register: fuse_nodes
+- name: find a worker node to label for loading tuned module
+  shell: oc get nodes -l node-role.kubernetes.io/worker -oname | head -1
+  when: fuse_nodes.stdout == ""
+  register: worker_nodes
+- name: mark node with our tuned label
+  command: oc label "{{ worker_nodes.stdout }}" "{{ tuned_module_fuse_label }}="
+  when: fuse_nodes.stdout == ""
+
+- name: apply tuned fuse profile manifest
+  command: oc apply -f "{{ tuned_module_fuse }}"
+
+- name: apply ci utils manifest
+  command: oc apply -f "{{ gpu_operator_ci_utils }}"
+
+- name: authorize ns/gpu-operator-ci Pods to access ns/gpu-operator-ci-utils images
+  command: oc policy add-role-to-user \
+             system:image-puller system:serviceaccount:gpu-operator-ci:default \
+             --namespace=gpu-operator-ci-utils
+
+- name: apply namespace manifest
+  command: oc apply -f "{{ gpu_operator_namespace }}"
+
+- name: apply imagestream manifest
+  command: oc apply -f "{{ gpu_operator_imagestream }}"
+
+- name: apply image helper builder manifest
+  command: oc apply -f "{{ gpu_operator_helper_image_builder }}"
+
+- name: wait for the helper image to be built
+  command: oc logs -f bc/helper-image-builder -n gpu-operator-ci-utils
```

Comment:
I am not a big fan of this approach, is not how ansible checks or any ci are meant to be. what if the logs are just... random strings on an endless loop? 
We should be doing something like 
```bash
OPERATOR_IMAGE=$(oc get pods --field-selector=status.phase=Succeeded -n gpu-operator-ci --no-headers 2> /dev/null | wc -l)
exec test $OPERATOR_IMAGE -eq 1
```

Reply:
sure, I'll update that